hummingbot/connector/exchange/kucoin/kucoin_exchange.pyx

```diff
@@ -412,14 +411,15 @@ cdef class KucoinExchange(ExchangeBase):
 
     async def _update_balances(self):
         cdef:
-            str path_url = "/api/v1/accounts?type=trade"
+            str path_url = CONSTANTS.ACCOUNTS_PATH_URL
             str asset_name
             set local_asset_names = set(self._account_balances.keys())
             set remote_asset_names = set()
 
-        data = await self._api_request("get", path_url=path_url, is_auth_required=True)
-        if data:
-            for balance_entry in data["data"]:
+        data = {"type": "trade"}  # this GET request requires that arguments be submitted as data
+        response = await self._api_request("get", path_url=path_url, data=data, is_auth_required=True)
```

Comment:
I tried sending the params as params instead of body, but was getting `Invalid KC-API-Sign` back from the server. When signing, the params should be passed as body and then encoded in the signature as well. I originally wanted to separate the parameters from the path url, but it's true that passing them directly into the url circumvents this issue, so I will return it back to the original configuration